### PR TITLE
added functionality to skip extracting already extracted archives ``-…

### PR DESCRIPTION
**Pull request by andox**
…-skip-if-exists``.

### CODE REVIEW

File: unrarall [hunk 1]
```diff
@@ -165,6 +167,40 @@ function getUnrarFlags()
   esac
 }
 
+#function to check if an archive have been extracted already
+function isAlreadyExtracted()
+{
+  case "$1" in
+    unrar)
+      contains=($(unrar lb "$2"))
+      for x in $contains; do
```
Comment (delcypher) [inline unrarall]: The indentation here looks kind of messed up.

File: unrarall [hunk 2]
```diff
@@ -165,6 +167,40 @@ function getUnrarFlags()
   esac
 }
 
+#function to check if an archive have been extracted already
+function isAlreadyExtracted()
+{
+  case "$1" in
+    unrar)
+      contains=($(unrar lb "$2"))
```
Comment (delcypher) [inline unrarall]: I don't have any rar files to hand to test this. Does `unrar lb` spit out the correct paths for where we extract them?

File: unrarall (hunk 1 above)
Comment (andox) [inline unrarall]: Hey!

Yeah sorry, indentation is probably incorrect, used my Macbook (which i do not use for coding normally) while kids was sleeping (was on parental leave)  ;)
Please feel free to change anything :)

**Comment by ashleyconnor** [comment]
I started using this branch and `cksfv` runs before the exists check. I think it should probably run after.

**Comment by arfoll** [comment]
Closing as #40 is going to be merged instead and provides the same functionality

**Pull request closed without merging**